pkg/cmd/pr/checkout/checkout.go

```diff
@@ -138,16 +141,16 @@ func checkoutRun(opts *CheckoutOptions) error {
 }
 
 func cmdsForExistingRemote(remote *context.Remote, pr *api.PullRequest, opts *CheckoutOptions) [][]string {
-	var cmds [][]string
-
 	remoteBranch := fmt.Sprintf("%s/%s", remote.Name, pr.HeadRefName)
 
 	refSpec := fmt.Sprintf("+refs/heads/%s", pr.HeadRefName)
 	if !opts.Detach {
 		refSpec += fmt.Sprintf(":refs/remotes/%s", remoteBranch)
 	}
 
-	cmds = append(cmds, []string{"git", "fetch", remote.Name, refSpec})
```

Comment:
If you didn't have a strong reason to edit this invocation, then please put it back the way it was. We prefer functional changes to not be bundled with stylistic ones.

Reply:
Np. I like that rule. 👍